src/middlewared/middlewared/alert/source/volume_version.py

```diff
@@ -17,8 +17,9 @@ def check_sync(self):
             if not self.is_upgraded(pool):
                 alerts.append(Alert(
                     "New feature flags are available for volume %s. Refer "
-                    "to the \"Upgrading a ZFS Pool\" section of the User "
-                    "Guide for instructions.",
+                    "to the \"Installing and Upgrading/Upgrading/ "
+                    "Upgrading a ZFS Pool\" section of the User Guide for "
+                    "instructions.",
```

Comment:
That will read "Installing and Upgrading/Upgrading/  a ZFS Pool"

Didnt you mean "Installing and Upgrading a ZFS Pool"?

Reply:
I'm trying to provide the full  Chapter/Section/Subsection path to the Upgrading a ZFS Pool:
Installing and Upgrading (Chapter)/Upgrading (Section)/Upgrading a ZFS Pool (Subsection)

Reply:
Let me rethink this string and try again - just a sec.

Reply:
Is the extra space intended, then?

"Installing and Upgrading/Upgrading/ Upgrading a ZFS Pool"

vs

"Installing and Upgrading/Upgrading/Upgrading a ZFS Pool"